reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -1,35 +1,57 @@
 .heatmap {
   overflow-y: auto;
+  overflow-x: auto;
   max-height: 600px;
   height: 100%;
   background-color: #ffffff;
 }
 
-.heatmap table {
-  overflow-y: auto;
+.heatmap-container {
+  padding: 0;
 }
 
 .heatmap table th {
   vertical-align: top;
   position: sticky;
   top: 0;
   z-index: 2;
-  background-color: white;
+  background-color: inherit;
   color: #133a53;
 }
 
+.darkmode .table thead th {
```

Comment:
the selector should be the same as the lightmode block it is overriding (with .darkmode prefix)

Reply:
✅